canonicalwebteam/discourse/parsers/base_parser.py

```diff
@@ -610,9 +610,6 @@ def _replace_image_src(self, soup):
         Given some HTML soup, replace relative image srcs
         """
         for img in soup.findAll("img"):
-            src = img.get("src", "")
```

Comment:
I specifically added this previously for fixing relative images from discourse - removing this will break other images. I think the correct fix would be to do:
`if src and src.startswith("/") and not src.startswith("//")` on the line below

`//` is just a shorthand to use the same protocol as the current page.

Reply:
Ohhh, Ok. Will fix that.